src/common/Header/index.js

```diff
@@ -1,9 +1,42 @@
-import { Stroke1, Wrapper } from "./styled";
+import {
+    Wrapper,
+    StyledHeader,
+    TittleWrapper,
+    Search,
+    SearchWrapper,
+    PageTittle,
+    Menu,
+    MenuItem,
+    SearchIcon,
+    Img,
+    Input,
+} from "./styled";
 
 export const Header = () => {
     return (
         <Wrapper>
-        <Stroke1></Stroke1>
+            <StyledHeader>
+                <TittleWrapper>
+
+                        <img src='https://i.postimg.cc/pdVjVLc2/Video.png' alt="camera picture" />
```

Comment:
Może lepiej wrzucić to na githuba w razie gdyby link z hostingu w przyszłości wygasł? Może tę ikonkę będzie lepiej wyeksportować do svg?